sirepo/template/template_common.py

```diff
@@ -230,7 +230,7 @@ def validate_model(model_data, model_schema, enum_info):
                 'BeamList', 'MirrorFile', 'ImageFile', 'String', 'OptionalString', 'MagneticZipFile',
```

Comment:
The list is for string types that need to be escaped. Others are parsed differently so should not be in this list.